### ISSUE TITLE
Integrate QR code generator with new setup

### ISSUE THREAD

**Issue by BobrImperator** [blogic/ucentral-pwa#38]
Once #6 is finished, #27 can be used to implement the 2 last screens of new setup wizard.

https://share.goabstract.com/283bc06b-083d-4cbc-99fb-fc7304b12383?mode=design&sha=c609991f3db28c212d5b7a820c3a81423b3681a6

https://share.goabstract.com/837c95b8-2083-471e-b09e-7d2df8090914?mode=design&sha=c609991f3db28c212d5b7a820c3a81423b3681a6

# Questions
- [ ] While setting up network, user provides only name and password while the QR generator rquires 'encryption' as well, where do we get that from? Might be related to #36 

# TODOS
- [ ] Use Uc::NetworkQrCode component to generate the newly setup network with new setup wizard
- [ ] Add step definitions to the new setup template
- [ ] Implement translations
- [ ] It will be required to run a 'goToStep' callback in submitTask to redirect to the new steps.

**Comment by marcoow** [comment]
@BobrImperator I believe this is a duplicate of #29

**Comment by BobrImperator** [comment]
Yes, it is, I didn't realize we had an issue for that back then @marcoow.
Feel free to close this one.
